stacks-coordinator/src/bitcoin_node.rs

```diff
@@ -169,6 +169,7 @@ impl LocalhostBitcoinNode {
             .map(|path| Cow::Owned(format!("{}{}", &self.bitcoind_api, path)))
```

Comment:
Nit: Just pasting @stjepangolemac's suggestion here to make it easier to commit.
```suggestion
            .map(|path| {
                Cow::Owned(format!(
                    "{}/{}",
                    &self.bitcoind_api,
                    path.trim_start_matches('/')
                ))
            })
            .unwrap_or(Cow::Borrowed(&self.bitcoind_api));
```